app/src/main/res/layout/row_remote_install.xml

```diff
@@ -25,7 +25,7 @@
       android:layout_marginTop="20dp"
       android:text="BRAVIA 2015"
       android:textAppearance="?android:attr/textAppearanceMedium"
-      android:textColor="#ff6600"
+      android:textColor="?attr/colorPrimaryDark"
```

Comment:
This line crashes on my device (Pixel 3) -> inflate exception. This happens because the context being passed to the LayoutInflator does not have the correct theme applied. Either pass the activity context to it or try using the ContextThemeWrapper with ThemeManager (should solve this issue)